src/main/kotlin/uk/gov/dwp/dataworks/Application.kt

```diff
@@ -7,5 +7,6 @@ import org.springframework.boot.autoconfigure.SpringBootApplication
 @SpringBootApplication
 class Application
 fun main(args: Array<String>) {
+
```

Comment:
can you remove this new line so that the file isn't needlessly altered, makes the history much easier to read